### PR TITLE
Ros2 v0.8.0 pointcloud preprocessor

### PR DESCRIPTION
**Pull request by wep21**
### Note
- depends on https://github.com/tier4/AutowareArchitectureProposal.iv/pull/291

### CODE REVIEW

File: sensing/preprocessor/pointcloud/pointcloud_preprocessor/package.xml [hunk 1]
```diff
@@ -29,11 +29,13 @@
   <depend>pcl_ros</depend>
   <depend>libopencv-dev</depend>
   <depend>libpcl-all-dev</depend>
+  <depend>autoware_utils</depend>
+  <depend>lanelet2_extension</depend>
 
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_cmake_cppcheck</test_depend>
```
Comment (k0suke-murakami) [inline sensing/preprocessor/pointcloud/pointcloud_preprocessor/package.xml:36]: Need to include `ament_lint_common` like other pakcages? https://github.com/wep21/autoware.iv.universe/blob/9232f7117f8ee0c266e354ceba80feb41ab2137d/system/autoware_state_monitor/package.xml#L32

**Review by k0suke-murakami — CHANGES_REQUESTED** [review]
Could not build since could not resolve dependency of `autoware_utils` even after #291 is merged
```
In file included from /home/kosuke/autoware_0_8/src/autoware/pilot.auto/sensing/preprocessor/pointcloud/pointcloud_preprocessor/src/vector_map_filter/lanelet2_map_filter_nodelet.cpp:18:
/home/kosuke/autoware_0_8/src/autoware/pilot.auto/sensing/preprocessor/pointcloud/pointcloud_preprocessor/include/pointcloud_preprocessor/vector_map_filter/lanelet2_map_filter_nodelet.hpp:27:10: fatal error: autoware_utils/geometry/boost_geometry.hpp: No such file or directory
   27 | #include "autoware_utils/geometry/boost_geometry.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```

**Comment by wep21** [comment]
@k0suke-murakami ready for review

**Review by k0suke-murakami — APPROVED** [review]

**Comment by k0suke-murakami** [comment]
- [x]  Build passed

- [x] ros2 run success.
 Node : pointcloud_accumulator_node, concatenate_data_node, ray_ground_filter_node, crop_box_filter_node, ring_outlier_filter_node, distance_based_compare_map_filter_node , voxel_based_approximate_compare_map_filter_node ,downsample_filter_node, voxel_based_compare_map_filter_node, passthrough_filter_node , voxel_distance_based_compare_map_filter_node, passthrough_filter_uint16_node  ,voxel_grid_outlier_filter_node
- [x] Test  passed

**Pull request merged**